content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,175 @@
+a `Service` as a `parentRef`, to control how traffic to specific services behave.
+For more information, read the 
+[service mesh](https://gateway-api.sigs.k8s.io/mesh/) documentation or see the
+list of 
+[implementations](https://gateway-api.sigs.k8s.io/implementations/#service-mesh-implementation-status).
+
+Leading Contributors: Flynn, John Howard, Keith Mattix, and Mike Morris
+
+#### [GRPCRoute](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+If you are already using the experimental version of GRPCRoute, we recommend holding
+off on upgrading to the standard channel version of GRPCRoute until the
+controllers you're using have been updated to support GRPCRoute v1. Until then,
+it is safe to upgrade to the experimental channel version of GRPCRoute in v1.1
+that includes both v1alpha2 and v1 API versions.
+
+Leading Contributor: Richard Belleville
+
+#### [ParentRef Port](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.ParentReference)
+Now you can use the `port` field in ParentRefs to attach resources to Gateways,
+Services, or other parent resources. For example, you can attach an HTTPRoute to
+one or more specific Listeners of a Gateway based
+on the Listener `port`, instead of `name` field.
+
+Leading Contributor: Frank Budinsky
+
+#### [Conformance Profiles and Reports](https://gateway-api.sigs.k8s.io/concepts/conformance/#conformance-profiles)
+
+The Conformance report API has been expanded with the `mode` field (intended to
+specify the working mode of the implementation), and the `gatewayAPIChannel`
+(standard or experimental). The `gatewayAPIVersion` and `gatewayAPIChannel` are
+now filled in automatically by the suite machinery, along with a brief
+description of the testing outcome. The Reports have been reorganized in a more
+structured way, and the implementations can now add information on how the tests
+have been run and provide reproduction steps.
+
+Leading Contributors: Christine Kim, Mattia Lavacca, Shane Utt
+
+### New Additions to Experimental Channel
+
+#### [Gateway Client Certificate Verification]()
+
+Gateways can now configure client cert verification for each Gateway Listener by
+introducing a new field `frontendValidation` field within `tls`. This field
+supports configuring a list of CA Certificates that can be used as a trust
+anchor to validate the certificates presented by the client.
+
+Leading Contributor: Arko Dasgupta
+
+#### [Session Persistence and BackendLBPolicy]()
+
+Session Persistence is being introduced to Gateway API via a new policy
+(BackendLBPolicy) for Service-level configuration and as fields within HTTPRoute
+and GRPCRoute for route-level configuration. The BackendLBPolicy and route-level
+APIs provide the same session persistence configuration, including session
+timeouts, session name, session type, and cookie lifetime type.
+
+Leading Contributors: Grant Spence, Gina Yeh
```

Comment:
@gcs278 @ginayeh please check and add/change anything you'd like via code suggestion.

Reply:
Seems like folks are providing examples, I can do that here too if we think appropriate for session persistence. I could add a route-level example too, but may be a bit much.

```suggestion

Below is an example configuration of `BackendLBPolicy` that enables cookie-based session persistence
for the `foo` service. It sets the session name to `foo-session`, defines absolute and idle timeouts, and configures the cookie to be a session cookie:

~~~
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: BackendLBPolicy
metadata:
  name: lb-policy
  namespace: foo-ns
spec:
  targetRefs:
  - group: core
    kind: service
    name: foo
  sessionPersistence:
    sessionName: foo-session
    absoluteTimeout: 1h
    idleTimeout: 30m
    type: Cookie
    cookieConfig:
      lifetimeType: Session
~~~

Leading Contributors: Grant Spence, Gina Yeh
```